chef_master/source/resource_registry_key.rst

```diff
@@ -24,11 +24,7 @@ A **registry_key** resource block creates and deletes registry keys in Microsoft
 .. code-block:: ruby
 
    registry_key 'HKEY_LOCAL_MACHINE\\...\\System' do
-     values [{
```

Comment:
I find the original form here a lot easier to understand. It's super clear that we're setting these 3 hash values.

Reply:
That's fine. I was trying to make sure it matched the change to the vscode plugin I had a PR in for that was approved. Was afraid it may cause confusion. No worries